aptos-move/move-examples/mint_nft/step3/sources/nft_step3.move

```diff
@@ -0,0 +1,231 @@
+/// This module is the step 3 of our NFT Move tutorial, building on top of step 2. In this module, we introduce an admin account and
+/// admin functions to update the configuration of this module.
+///
+/// We need admin accounts, because we give up control over the smart contracts when we deploy a module using a resource account.
+/// Sometimes that's okay, but there are times when we still wanted to remain some control over the module - for example,
```

Comment:
Replace "still wanted to remain" with "still want to retain"

Reply:
yeah that sounds much better!!